### ISSUE TITLE
Export -> Import .zip file fail

### ISSUE THREAD

**Issue by Cmdv** [quasar-analytics/quasar#2652]
### Description

Downloading a file in `.zip` format, when uploading that file you get given an error.

### Reproduction guide

on Mongo: download `smallZips` selecting `.zip` -> upload that zip `smallZips.zips` -> Error message

### Results
Here are screen shots of the error and network headers etc.

<img width="472" alt="screen shot 2017-08-18 at 12 07 21" src="https://user-images.githubusercontent.com/3036557/29456906-af4a508e-840f-11e7-9cbe-aff04fcdabb3.png">
<img width="745" alt="screen shot 2017-08-18 at 12 13 13" src="https://user-images.githubusercontent.com/3036557/29456915-b442eb0a-840f-11e7-9636-7a19cde0d656.png">

**Comment by garyb** [comment]
According to the docs this is the "expected" behaviour, as zip uploading seems to only be intended for the [multiple files case](https://github.com/quasar-analytics/quasar#uploading-multpile-files)... so this is an enhancement request rather than a bug report as such.

I guess ideally we would want to support the zipped-file upload by allowing zips to be accepted for the `PUT /data/fs/[file]` path (rather than just dir paths), and then use the provided headers to determine the data format, and then when the zip is received, reject it if it contains more than one file.

Basically what I'm getting at, is enhancing the current file upload support by allowing a `application/zip,application/ldjson;mode=precise` style header for the file upload, similar to the way we handle it in the download.

**Comment by nicflores** [comment]
The archive metadata is missing from the zip archive downloaded. PR #2971 will fix this.
